src/cli/dev.js

```diff
@@ -6,7 +6,6 @@
  * Side Public License, v 1.
  */
 
-require('@kbn/babel-register').install();
```

Comment:
__note:__ `require('@kbn/babel-register').install()` is already done inside of `setup_node_env`.